Cargo.lock

```diff
@@ -2419,19 +2795,21 @@ dependencies = [
 
 [[package]]
 name = "object_store"
-version = "0.5.0"
+version = "0.5.3"
```

Comment:
dummy question: while datafusion 14 uses object_store 0.5.0, why Cargo is allow to upgrade to 0.5.3?

https://github.com/apache/arrow-datafusion/blob/14.0.0/datafusion/core/Cargo.toml#L81

Reply:
pinning to 0.5.0 is equivalent to pinning to 0.5.x i believe.

Reply:
oh, right, I should take detail look at it (was from Ruby background and thought they are similar, turn out NOT)

https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html

```
0.2.3  :=  >=0.2.3, <0.3.0
1.2.3  :=  >=1.2.3, <2.0.0
```